includes/Core/Util/Activation.php

```diff
@@ -155,9 +155,81 @@ private function get_activation_notice() {
 					ob_start();
 					?>
 					<script type="text/javascript">
-					if( 'undefined' !== typeof sendAnalyticsTrackingEvent ) {
-						sendAnalyticsTrackingEvent( 'plugin_setup', 'plugin_activated' );
-					}
+						if( 'undefined' !== typeof sendAnalyticsTrackingEvent ) {
```

Comment:
Does `if(` not fail eslint? 🤔

Reply:
Maybe not because it's in a PHP file? I'll fix it, in any case.